lib/buffer.js

```diff
@@ -98,8 +100,6 @@ Buffer.from = function(value, encodingOrOffset, length) {
   return fromObject(value);
 };
 
-Object.setPrototypeOf(Buffer, Uint8Array);
```

Comment:
Yes, it does. I can't remember the reason for this change, I think I confused it with something else. I'll remove it.